diff --git a/node_build/make.js b/node_build/make.js
index 6f25afa..d08800d 100644
--- a/node_build/make.js
+++ b/node_build/make.js
@@ -56,7 +56,7 @@ Builder.configure({
         '-std=c99',
         '-Wall',
         '-Wextra',
-        '-Werror',
+//        '-Werror',
         '-Wno-pointer-sign',
         '-pedantic',
         '-D',builder.config.systemName + '=1',
@@ -66,9 +66,9 @@ Builder.configure({
         // Broken GCC patch makes -fstack-protector-all not work
         // workaround is to give -fno-stack-protector first.
         // see: https://bugs.launchpad.net/ubuntu/+source/gcc-4.5/+bug/691722
-        '-fno-stack-protector',
-        '-fstack-protector-all',
-        '-Wstack-protector',
+//        '-fno-stack-protector',
+//        '-fstack-protector-all',
+//        '-Wstack-protector',
 
         '-D','HAS_BUILTIN_CONSTANT_P',
 
@@ -84,9 +84,10 @@ Builder.configure({
 
         // disable for speed, enable for safety
         '-D','Identity_CHECK=1',
-        '-D','Allocator_USE_CANARIES=1',
-        '-D','PARANOIA=1'
+        '-D','Allocator_USE_CANARIES=1'
+       // '-D','PARANOIA=1'
     );
+    process.env['Log_LEVEL'] = 'INFO';
     var logLevel = process.env['Log_LEVEL'] || 'DEBUG';
     builder.config.cflags.push('-D','Log_'+logLevel);
     if (process.env['NO_PIE'] === undefined) {
